x-pack/plugins/infra/server/lib/host_details/process_list_chart.ts

```diff
@@ -29,6 +29,7 @@ export const getProcessListChart = async (
               [timefield]: {
                 gte: to - 60 * 1000, // 1 minute
                 lte: to,
+                format: 'epoch_millis',
```

Comment:
This is called when showing the details of a process of a node in the Metrics waffle chart